dali/kernels/imgproc/warp/block_warp.cuh

```diff
@@ -56,6 +56,40 @@ __device__ void BlockWarp(
   }
 }
 
+
+template <DALIInterpType interp_type, typename Mapping,
+          int ndim, typename OutputType, typename InputType,
+          typename BorderType>
+__device__ void BlockWarp(
+    SampleDesc<3, OutputType, InputType> sample, BlockDesc<3> block,
```

Comment:
```suggestion
    SampleDesc<ndim, OutputType, InputType> sample, BlockDesc<ndim> block,
```
? Otherwise ndim is not used at all. Also it should probably be spatial_ndim

Reply:
It should be removed.